### PR TITLE
Add only index.js at a-tiny-ls-world

### PR DESCRIPTION
**Pull request by alexdefender**
Sorry, I deleted previous branch and request 221 was deleted. But I added new with one index.js file.

### task name

[Demo](https://alexdefender.github.io/a-tiny-JS-world/) |
[Code base](https://github.com/alexdefender/a-tiny-JS-world)

The code is submitted in a dedicated feature branch.
Please, review.

### CODE REVIEW

File: submissions/alexdefender/a-tiny-JS-world/index.js [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+    for (let index in obj) {
+        print(index + ': ' + obj[index]);
+    }
+}
+
+printInfo(dog);
+printInfo(cat);
+printInfo(man);
+printInfo(woman);
+printInfo(catWoman);
```
Comment (OleksiyRudenko) [inline submissions/alexdefender/a-tiny-JS-world/index.js]: Ensure every code file ends with an empty line. Tune up your IDE to this dirty job for you.

File: submissions/alexdefender/a-tiny-JS-world/index.js [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+   print('human; John; male; 2; 2; Hello world!; Rex, Tom, Jenny');
+   print('human; <strong>John</strong>; male; 2; 2; <em>Hello world!</em>; Rex, Tom, Jenny');
+   print('human; <strong>John</strong>; male; 2; 2; <em>Hello world!</em>; Rex, Tom, Jenny', 'div');
+   */
+
+function printInfo(obj) {
+    print('--- Object ---');
+    for (let index in obj) {
+        print(index + ': ' + obj[index]);
+    }
```
Comment (OleksiyRudenko) [inline submissions/alexdefender/a-tiny-JS-world/index.js]: Please, change this function so the output follows tech specs examples.
Order of properties in objects is not guaranteed. Please, list properties to include into output string explicitly.
Please, employ `Array` methods.

File: submissions/alexdefender/a-tiny-JS-world/index.js (hunk 1 above)
Comment (OleksiyRudenko) [inline submissions/alexdefender/a-tiny-JS-world/index.js]: Imagine you have 10s of objects. Please, optimize this fragment using an appropriate `Array` method.

File: submissions/alexdefender/a-tiny-JS-world/index.js [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+const woman = entity({}, 'human', 'Elena', 'female', 2, 2, 'I am a woman!', ['Anna', 'Jon']);
+const man = entity({}, 'human', 'Alex', 'male', 2, 2, 'I am a man!', ['Tim', 'Petr']);
+const catWoman = entity({}, 'human', 'Eleonora', 'female', 2, 2, cat.saying, ['Super-man', 'Spider-man']);
+
+var entityes = [dog, cat, woman, man, catWoman];
+
+entityes.forEach(element => {
+    print(element.species + ', ' + element.name + ', ' + element.gender + ', ' +
+    element.legs + ', ' + element.hands + ', ' + element.saying + '\n');
+});
```
Comment (OleksiyRudenko) [inline submissions/alexdefender/a-tiny-JS-world/index.js]: There is a pattern in how you build a string for output.
Please, use a dedicated function for that, as this is an important job and worth being a reusable function.
Also consider the following example:
``[aVariable, obj.property, "string", `template string with ${aVariable} embedded`].join("; ");``


P.S. Please, fix issue with ending empty line. It is important.

**Comment by alexdefender** [comment]
Done

**Pull request merged**